syntax = "proto3";

package mprpc; // 定义命名空间

message RpcHeader
{
    bytes service_name = 1; //  服务名称
    bytes method_name = 2;  // 方法名称
    uint32 args_size = 3;  //  消息长度
}